src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests.csproj

```diff
@@ -26,6 +26,10 @@
     <InternalsVisibleToMoq Include="DynamicProxyGenAssembly2" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Dependencies\VisualStudio\VisualStudio.csproj">
```

Comment:
This shouldn't have a reference to any Visual Studio piece; this is host-agnostic.